1

```diff
@@ -0,0 +1,11 @@
+Make UnrecoverableS3OperationException unchecked
```

Comment:
You were too fast to comment :) I removed it almost immediately after I pushed it.